src/chapters/typescript/exercises.rst

```diff
@@ -0,0 +1,178 @@
+Exercises: TypeScript
+=====================
+
+Part 1 - Declare Variables With Type
+------------------------------------
+
+#. Fork the `part 1 repl.it <https://repl.it/@launchcode/ts-exercises-part-1-and-2>`_
+#. Declare and assign a variable for each in the below table.
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Variable Name
+     - Type
+     - Value
+   * - spacecraftName
+     - string
+     - ``'Determination'``
+   * - speedMph
+     - number
+     - 17500
+   * - kilometersToMars
+     - number
+     - 225000000
+   * - kilometersToTheMoon
+     - number
+     - 384400
+   * - milesPerKilometer
+     - number
+     - 0.621
+
+
+Part 2 - Print Days to Mars
+---------------------------
+In the *same* repl.it you completed Part 1, do the following.
+
+#. Declare and assign these variables
+
+   * Each variable should it's type declared
```

Comment:
"Remember: variable declarations in TypeScript include the type!"

Reply:
Changed.